api/test/media_integration.py

```diff
@@ -103,6 +103,16 @@ def search_consistency(
             results.add(media_id)
 
 
+def search_sort(media_type, sort_dir, exp_created_on):
+    response = requests.get(
+        f"{API_URL}/v1/{media_type}/?page_size=1&unstable__sort_by=indexed_on&unstable__sort_dir={sort_dir}",
```

Comment:
There is some bug in the dead-link filtering + pagination code that is causing the items in the output to be moved around. Setting `page_size=1` and iterating over the pages yields elements in the correct sorted order but any higher page size and the order get messed up slightly.

Reply:
I'll have to fire up a debugger and go over it line by line but it's late and I don't have the mental energy to go into that area of the API code.

Reply:
Ah I misjudged the location of the bug, it was in the ES to DB mapping. Fixed in 1839145.